AppDB/appscale/datastore/backup/cassandra_backup.py

```diff
@@ -238,4 +239,21 @@ def restore_data(path, keyname, force=False):
     utils.ssh(db_ip, keyname,
       'monit start {}'.format(CASSANDRA_MONIT_WATCH_NAME))
 
+  logging.info('Waiting for all nodes in cluster to start')
```

Comment:
This was a fix for the Migration job. In master, the tools persistently retry UAServer-related calls indefinitely, but the AdminServer just returns an error when unable to deploy. With https://github.com/AppScale/appscale-testing/pull/221, I don't think this change is needed, so I will remove it from this pull request.